src/index.js

```diff
@@ -20,6 +20,11 @@ if (typeof window !== 'undefined') {
   const AppWithRouter = withRouter(App);
   try {
     store.loginUser = JSON.parse(getCookie('loginUser') || '{}');
+    const changeDev = getCookie('changeDev');
```

Comment:
`changeDev` is a verb, rename it as noun

Reply:
ok, rename 'role'.